examples/src/main/java/io/dapr/examples/pubsub/http/CloudEventPublisher.java

```diff
@@ -47,7 +47,7 @@ public static void main(String[] args) throws Exception {
     //Creating the DaprClient: Using the default builder client produces an HTTP Dapr Client
     try (DaprClient client = new DaprClientBuilder().build()) {
       for (int i = 0; i < NUM_MESSAGES; i++) {
-        CloudEvent cloudEvent = new CloudEvent();

```

Comment:
Is this required? Changes must be backwards compatible, which means that code without generics should still work as-is.

Reply:
It is not required, in the sense that it will also work without specifying the generic type. You can see that in the `PubSubIT`, where even the published message is declared without a generic type. On the consuming side, it can be consumed as a `CloudEvent` or as a `CloudEvent<PubSubIT.MyObject>`. See `SubscriberController` for an example of that.

I thought it would make sense to update the documentation. Since this file is in the **examples/** folder, I consider it part of the documentation, and hence decided to update it.